README.md

```diff
@@ -11,6 +11,14 @@
 - Java 1.8 JDK
 	- Older versions of Java can be downloaded at oracle.com/java/technologies/downloads/archive/ (requires an Oracle account)
 
+## Clone the repo
+```bash
+cd <directory you want to download into>
+git clone --recurse-submodules https://github.com/RC2-Tools/RC2-desktop.git
```

Comment:
I don't think you need the URL in the main version since it's embedded in the submodule file itself.

Reply:
Can you try it out and let me know. (I agree needed for your forked version)